libkbfs/tlf_journal.go

```diff
@@ -416,12 +416,26 @@ func (j *tlfJournal) flush(ctx context.Context) (err error) {
 	return nil
 }
 
+var errTLFJournalShutdown = errors.New("tlfJournal is shutdown")
+
+func (j *tlfJournal) checkShutdownLocked() error {
+	if j.blockJournal == nil || j.mdJournal == nil {
+		return errTLFJournalShutdown
+	}
+	return nil
+}
+
 func (j *tlfJournal) getNextBlockEntryToFlush(ctx context.Context,
 	currentUID keybase1.UID, currentVerifyingKey VerifyingKey) (
 	journalOrdinal, *blockJournalEntry, []byte,
 	BlockCryptKeyServerHalf, error) {
 	j.journalLock.RLock()
 	defer j.journalLock.RUnlock()
+	err := j.checkShutdownLocked()
```

Comment:
Could save a few lines by combining the `check` call and the `if` statement into one line everywhere.